core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -5,17 +5,73 @@
         <span translate="I18N_FOOTER_FOLLOW_US"></span>:
       </h4>
     </div>
-    <a href="https://plus.google.com/109898456505810251700/about" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/gplus.png')]>" alt="Google+">
-    </a>
-    <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/youtube.png')]>" alt="YouTube">
-    </a>
-    <a href="https://www.facebook.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/fb.png')]>" alt="Facebook">
-    </a>
-    <a href="https://twitter.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/twitter.png')]>" alt="Twitter">
-    </a>
+    
+    <div class="oppia-footer-icons">
+
+        <div class="google-plus-color">
+            <a href="https://plus.google.com/109898456505810251700/about" target="_blank"><i class="fa fa-google-plus"></i>
+            <span class="oppia-icon-accessibility-label">Follow us at Google Plus</span>
+            </a>
+        </div>
+
+         <div class="youtube-color">
+           <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank"><i class="fa fa-youtube-play"></i>
+            <span class="oppia-icon-accessibility-label">Subscribe us at Youtube</span>
+          </a>
+        </div>
+
+        <div class="fb-color">
+            <a href="https://www.facebook.com/oppiaorg" target="_blank"><i class="fa fa-facebook"></i>
+              <span class="oppia-icon-accessibility-label">Like us at Facebook</span>
+            </a>
+        </div>
+        
+        <div class="twitter-color">
+            <a href="https://twitter.com/oppiaorg" target="_blank"><i class="fa fa-twitter"></i>
+                <span class="oppia-icon-accessibility-label">Follow us at Twitter</span>
+            </a>
+        </div>
+    </div>
+
   </div>
 </script>
+
+<style>
+  .oppia-footer-social .oppia-footer-icons {
+        display: -webkit-inline-box;
+        font-size: 24px;
+        position: relative;
+        top: -10px;
+    }
+
+    .oppia-footer-icons .google-plus-color {
+        background: #dd4b39;
+        margin-right: 6px;
+        padding: 5px 7px;
+    }
+    .oppia-footer-icons .youtube-color {
+        background: #cd201f;
+        margin-right: 6px;
+        padding: 5px 9px;
+    }
+    .oppia-footer-icons .fb-color {
+        background: #3b5998;
+        margin-right: 6px;
+        padding: 5px 10px;
+    }
+    .oppia-footer-icons .twitter-color {
+        background: #1da1f2;
+        margin-right: 6px;
+        padding: 5px 6px;
+    }
+    .fb-color i.fa.fa-facebook {
```

Comment:
Not sure about these class names, as we normally namespace our CSS to prevent possible clashes with third-party styles.